CMakeLists.txt

```diff
@@ -128,7 +128,7 @@ if (${LLVM_VERSION_NUMBER} VERSION_GREATER "3.3.0")
 endif()
 
 if (ARM_ENABLED)
```

Comment:
Could you set ARM_ENABLED to ON by default so we would be able to check the builds in CI.